com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ClientReleaser.java

```diff
@@ -15,23 +15,23 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ClientReleaser.class);
 
-    private final FlowableEngineEvent event;
     private final CloudControllerClientProvider clientProvider;
+    private final HistoryService historyService = Context.getProcessEngineConfiguration()

```

Comment:
Why this is moved up here? If you do not want to create it every time, maybe passing it to the class would be better.